_includes/footer.html

```diff
@@ -1,12 +1,63 @@
-<footer class="usa-footer usa-footer-medium usa-section background-gray-dark" role="contentinfo">
+<footer class="usa-footer usa-section background-medium" role="contentinfo">
   <div class="usa-grid">
-    <h2>Drop us a line.</h2>
-    <p>To share your thoughts, email us anytime at <a href="mailto:{{ site.footer.email }}">{{ site.footer.email }}</a>.</p>
+    <div class="footer-logo-links">
+    {% include icons/oes.svg %}
```

Comment:
This is the link that pointed at the OES icon at https://github.com/gsa-oes/office-of-evaluation-sciences/blob/add-new-footer/_includes/icons/oes.svg

(see how it says "include icons / oes.svg?")